paprika/restraints/utils.py

```diff
@@ -25,4 +25,143 @@ def parse_window(window):
         raise Exception("Cannot determine the phase for this restraint.")
     window = int(window[1:])
 
-    return window, phase
\ No newline at end of file
+    return window, phase
+
+
+def restraints_from_ascii(filename):
+    """
+    Utility function to read in restraints from a simple ASCII file.
+
+    Parameters
+    ----------
+    filename : str
+        file name of template file.
```

Comment:
I think towards the end I started to try to use `os.path` or `pathlib.Path` objects instead of strings for filenames. I think it's the preferred method nowadays for cross-platform support. See, for example: https://github.com/slochower/pAPRika/blob/1bfce3a2705f908b354a5a2b0c2bc5261e767cc8/paprika/setup.py#L51

It should be transparent to turn this into that.